articles/libraries/lock/v10/customization.md

```diff
@@ -123,6 +124,16 @@ Examples of `allowedConnections`:
 
 ![Lock - Allowed Connections](/media/articles/libraries/lock/v10/customization/lock-allowedconnections-social.png)
 
+### allowShowPassword {Boolean}
+
+This option determines whether or not add a checkbox to the UI which, when selected, will allow the user to show their password when typing it. The option defaults to `false`.

```

Comment:
@jeffreylees You are missing a `to` here: `This option determines whether or not to add`